pam_eye.c

```diff
@@ -7,45 +8,87 @@
 #include <security/pam_modules.h>
 
 #define USERAGENT "pam_eye/1.0"
-#define TIMEOUT   10L
+#define DEFAULT_TIMEOUT_MS 200
```

Comment:
why so small? As out of the box feature, I think library should be more tolerant to network issues and have more big timeout, I guess 2 seconds it's more convenient.